### PR TITLE
Cgi change

### PR DESCRIPTION
**Pull request by 0xCAF3D0OD**
NEW: Make little changes 
- Add theo's variable for the bin/php-cgi.
- Remove useless condition.
- Remove comments in php_handler

bisous

### CODE REVIEW

**Review by RobinBurri — APPROVED** [review]
OK

File: inc/cgi.hpp [hunk 1]
```diff
@@ -43,8 +43,6 @@ class CGI
 	std::string _query;
 	std::string _uri;
 
-	static std::string _path_php_binary;
-
```
Comment (t-h2o) [inline inc/cgi.hpp:47]: Why set this attribute as public ?
There is already a setter method

File: src/http/Response.cpp [hunk 2]
```diff
@@ -235,18 +235,14 @@ void
...
-	//	std::string cgi_path = _server_config.get("php-cgi").get<std::string>();
-	std::string cgi_path = "/Users/kdi-noce/goinfre/php/php-8.2.5/sapi/cgi/php-cgi";
 	if (request.get_has_query())
 		std::cout << "the query string is : " << req_map["Query"] << std::endl;
 
 	std::string cgi_file = "test/website" + get_key(req_map, "Path");
 	std::string cgi_query = get_key(req_map, "Query");
 	std::string output_cgi;
-	CGI			cgi(cgi_path, cgi_file, cgi_query);
+	CGI			cgi(CGI::_path_php_binary, cgi_file, cgi_query);
```
Comment (t-h2o) [inline src/http/Response.cpp:245]: It doesn't make sense to use `_path_php_binary` like that.
The goal is to use the `_path_php_binary` directly into `CGI`

**Review by t-h2o — COMMENTED** [review]
**not** OK

File: src/http/Response.cpp (hunk 2 above)
Comment (0xCAF3D0OD) [reply src/http/Response.cpp]: done [55838b3](https://github.com/t-h2o/webserv/pull/82/commits/55838b31e16c136e0d5ff6ceb178e0e75ec216e0)

**Review by t-h2o — APPROVED** [review]
❤️

**Review by RobinBurri — APPROVED** [review]
Ok

**Pull request merged**